crates/sui-indexer-alt-graphql/src/api/types/transaction_effects.rs

```diff
@@ -193,7 +193,7 @@ impl EffectsContents {
         let transaction_digest = content.digest()?;
         let timestamp_ms = content.timestamp_ms();
 
-        for edge in cursors.edges {
+        for edge in cursors.edges.iter() {
```

Comment:
spurious change?

Reply:
Yes thanks for catching this, i'll use `git checkout [branch] [filepath]` to revert changes like this in future.